### PR TITLE
Improves website accessibility score

### LINKED ISSUE #336: [other] Potential Contributions for to Resolve Accessibility score
### What would you like to share?

I just did a quick accessibility test on your site using [Perfbuddy](https://perfbuddy.com/) and Chrome Lighthouse audit. Some areas can be improved and could be a great first issue/PR for new contributors. 

Based on the audit, here are some suggested changes by the audit:

- [ ] Buttons do not have an accessible name
Screenshot (11)
- [ ] `<frame>` or `<iframe>` elements do not have a title
- [ ] Links do not have a discernible name
Screenshot (12)
- [ ] Background and foreground colors do not have a sufficient contrast ratio. (Seems to be flagging the purple text and the grey color for the buttons)
Screenshot (13)
- [ ] Lists do not contain only `<li>` elements and script supporting elements (`<script>` and `<template>`).
Screenshot (14)

These are very few tweaks and I think separating them into different issues would be a great first contribution for newer developers 😄. I can also tweak some of these as well or anything else involving accessibility.

### Additional information

The audit only caught any issues easily picked up through Lighthouse. A [hands-on accessibility audit](https://web.dev/how-to-review/) may be necessary to tackle more specific items.

### PR DESCRIPTION
**Pull request by VijayalakshmiGanesh**
## Fixes Issue

This PR fixes the following issues:

closes #336 

## Changes proposed

Fixed the following issues to improve website accessibility score from 79 to 97 by adding/updating **aria-label, title, color** where necessary
1. 
image
2. 
image
3. 
image
4. 
image

## Check List (Check all the boxes which are applicable) 

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Output verified using Chrome LightHouse Audit for both mobile and desktop.
image
image

### CODE REVIEW

**Comment by Chandraprakash-Darji** [comment]
@VijayalakshmiGanesh make the highlight text yellow. becuse on hashnode page it is used yellow

**Comment by VijayalakshmiGanesh** [comment]
Output Verified in WAVE tool
image

**Comment by Chandraprakash-Darji** [comment]
@VijayalakshmiGanesh Scroll to top button has not descriptive text, add that

**Comment by Chandraprakash-Darji** [comment]
@VijayalakshmiGanesh Great work

**Comment by Chandraprakash-Darji** [comment]
Also add the alt text for the image
image
it is on the hasnode page

File: src/components/home/Hero/index.jsx [hunk 1]
```diff
@@ -11,12 +11,12 @@ const Hero = () => {
         <h1 className='heading'>Inspire. Educate. Empower</h1>
         <p className={styles.tagLine}>
           This is a community which provides{' '}
-          <span className='highlight highlight--light'>
+          <span className='highlight highlight--yelow'>
```
Comment (Chandraprakash-Darji) [inline src/components/home/Hero/index.jsx:14]: fix the typo of yellow

File: src/styles/root.css [hunk 2]
```diff
@@ -4,12 +4,14 @@
...
+  /* --content: 222, 227, 234; #dee3ea */
+  --content: 230, 235, 239; /* #E6EBEF */
   --divider: 30, 45, 61; /* #1E2D3D */
   --base-100: 20, 20, 20; /* #141414 */
   --base-200: 1, 8, 14; /* #01080E */
   --base-gray: 80, 80, 80; /* #505050 */
-  --primary: 89, 66, 233; /* #5942e9 */
+  /* --primary: 89, 66, 233; #5942e9 */
+  --primary: 69, 57, 172; /* #4539AC */
   --primary-hover: 64, 41, 209; /* #4029d1 */
```
Comment (Chandraprakash-Darji) [inline src/styles/root.css:15]: don't play with primary color

Comment (Chandraprakash-Darji) [reply src/styles/root.css:15]: Just reset back to old, because color you have set is not vibrant

**Review by Chandraprakash-Darji — CHANGES_REQUESTED** [review]

File: src/styles/root.css (hunk 2 above)
Comment (VijayalakshmiGanesh) [reply src/styles/root.css:15]: Is there any other alternative color that we could use to improve the contrast here
 or would you prefer that we leave the both content and primary color as is for the time being?

File: src/components/home/Hero/index.jsx (hunk 1 above)
Comment (VijayalakshmiGanesh) [reply src/components/home/Hero/index.jsx]: Fixed. Kindly check

File: src/styles/root.css (hunk 2 above)
Comment (VijayalakshmiGanesh) [reply src/styles/root.css]: I have reverted the changes. please check

Comment (Chandraprakash-Darji) [reply src/styles/root.css]: Let;s let it be for time being

**Review by Chandraprakash-Darji — CHANGES_REQUESTED** [review]

File: src/components/links/UnstyledLink.jsx [hunk 3]
```diff
@@ -28,6 +28,7 @@ const UnstyledLink = React.forwardRef(
         {...(!href.startsWith('#') && {
           target: '_blank',
           rel: 'noopener noreferrer',
+          'aria-label': ariaLabel,
```
[the remark this answers is not in the document]
Comment (VijayalakshmiGanesh) [reply src/components/links/UnstyledLink.jsx]: Removed as per your comments. Please check

**Comment by AnuragThePathak** [comment]
@VijayalakshmiGanesh fix the error occurring on running the Nodejs action.

**Comment by VijayalakshmiGanesh** [comment]
> @VijayalakshmiGanesh fix the error occurring on running the Nodejs action.

Please check now

**Review by Chandraprakash-Darji — APPROVED** [review]

**Pull request merged**